src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="post"
```

Comment:
The `action` attribute in the form tag should specify a URL where the form data will be sent. Currently, it is set to 'post', which is incorrect. Consider changing it to a valid URL or leaving it empty if the form is handled by JavaScript.